tasks/task_patch.c

```diff
@@ -628,6 +633,21 @@ static bool apply_patch_content(uint8_t **buf,
       free(ret_buf);
       *buf  = patched_content;
       *size = target_size;
+
+      /* Show an OSD message */
+      if (show_notification)
+      {
+         const char *patch_filename = basename(patch_path);
```

Comment:
We cannot use `basename()` - it is not platform-independent.

We must use `path_basename()` instead.

Reply:
fixed.

Please when you have time take a look at this [other PR by me](https://github.com/libretro/snes9x/pull/255) .